sdk/lib/convert/json.dart

```diff
@@ -185,6 +226,43 @@ class JsonCodec extends Codec<Object?, String> {
 }
 
 /// This class converts JSON objects to strings.
+///
+/// Example:
+///
+/// ```dart
+/// const JsonEncoder encoder = JsonEncoder();
+/// var values = {};
+/// values['text'] = 'foo';
+/// values['value'] = 2;
+///
+/// var jsonData = {};
+/// jsonData['content'] = values;
+///
+/// final String jsonString = encoder.convert(jsonData);
+/// print(jsonString); // {"content":{"text":"foo","value":2}}
+///  ```
+///
+/// Multiline example:
+///
+/// ```dart
+/// const JsonEncoder encoder = JsonEncoder.withIndent('  ');
+///
+/// var values = {};
+/// values['text'] = 'foo';
+/// values['value'] = 1;
```

Comment:
Can be const and assigned directly as map literal

Reply:
updated.